build/MSBuildToolset/build-toolset.ps1

```diff
@@ -35,46 +47,58 @@ try
         Copy-Item -Force "$NuGetTaskDir\*" $NuGetExtensionsDir
 
         # Copy the portable targets
+        write-host "Copy portable targets"
         Copy-Item -Force -Recurse (Join-Path ([Environment]::GetFolderPath("ProgramFilesX86")) "MSBuild\Microsoft\Portable") "$extensionsMSDir"
-
+        
         # Copy over the portable build task
         # TODO(https://github.com/dotnet/roslyn/issues/9640)
         # Copy from a NuGet package instead of a build
-        foreach ($item in ("Microsoft.Build.Tasks.CodeAnalysis.dll","Microsoft.CSharp.Core.targets","Microsoft.VisualBasic.Core.targets"))
+        write-host "Copy portable build task"
+        foreach ($item in (
+            "CodeAnalysisBuildTask\Microsoft.Build.Tasks.CodeAnalysis.dll",
+            "CodeAnalysisBuildTask\Microsoft.CSharp.Core.targets",
+            "CodeAnalysisBuildTask\Microsoft.VisualBasic.Core.targets", 
+            "csccore\csc.exe", 
```

Comment:
@tmat Not sure what you mean by that. We need a csc.exe to compile the code. That essentially *is* the toolset, right?